src/shipit_static_analysis/shipit_static_analysis/clang/tidy.py

```diff
@@ -111,7 +111,17 @@ def run(self, revision):
             logger.error('Mach static analysis failed: {}'.format(e.output))
             raise
 
-        issues = self.parse_issues(clang_output.decode('utf-8'), revision)
+        clang_output = clang_output.decode('utf-8')
+
+        # Dump raw clang-tidy output as a Taskcluster artifact (for debugging)
+        clang_output_path = os.path.join(
+            settings.taskcluster_results_dir,
+            'clangtidy.txt',
```

Comment:
Please add a unique element to the path name (revision or review id for example)

Reply:
Oh, I assumed that Taskcluster already did something similar, because the debug `report.json` also gets [dumped like that](https://github.com/mozilla-releng/services/blob/124aa12c781de6af31de9a16896c6e9973242b71/src/shipit_static_analysis/shipit_static_analysis/report/debug.py#L25).

Reply:
There is only one `report.json` per run, whereas we could have several clang-tidy runs

Reply:
Aha, makes sense, thanks.

Reply:
Done, I added a `{revision}-` prefix.